LoopKitUI/Views/InsulinTypeChooser.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  InsulinTypeChooserView.swift
+//  MockKitUI
+//
+//  Created by Pete Schwamb on 12/30/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+import LoopKit
+
+public struct InsulinTypeChooser: View {
+    
+    @Binding private var insulinType: InsulinType
+    
+    let supportedInsulinTypes: [InsulinType]
+    
+    public init(insulinType: Binding<InsulinType>, supportedInsulinTypes: [InsulinType]) {
+        self.supportedInsulinTypes = supportedInsulinTypes
+        self._insulinType = insulinType
+    }
+
+    public var body: some View {
+        ForEach(supportedInsulinTypes, id: \.self) { insulinType in
+            HStack {
+                ZStack {
+                    Image(frameworkImage: "vial")
+                        .resizable()
+                        .scaledToFit()
+                    Image(frameworkImage: "vial_fg")
```

Comment:
Both of these images aren't in the assets folder (though there is one named 'vial_stroke_light'), so the icons don't render.